.github/workflows/build.yml

```diff
@@ -0,0 +1,56 @@
+name: build
+on:
+  workflow_dispatch:
+  push:
+    branches: [master, "release-**"]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        ghc: ["8.6.5", "8.8.4", "8.10.2"]
+    name: build - ${{ matrix.ghc }} - ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install LLVM (for running tests)
+        run: sudo apt-get -y install llvm
```

Comment:
This is not guaranteed to be a stable llvm version or the one that's first on the path. If it works, though, it works :)
But I would immediately be suspicious of it should things start failing due to llvm down the line (either from code changes or github runner changes)

Reply:
The fuzzing workflow I'm in the process of should help ensure that this part will work with any released version of LLVM, so I'm not really worried in this case. The use of LLVM in this particular workflow is minimal, and should be pretty robust to version shifts.